FunctionSignatureMap.php

```diff
@@ -3604,31 +3605,31 @@
 'get_call_stack' => [''],
 'get_called_class' => ['class-string'],
 'get_cfg_var' => ['string|false', 'option_name'=>'string'],
-'get_class' => ['class-string', 'object='=>'object'],
-'get_class_methods' => ['array<int,string>', 'class'=>'mixed'],
+'get_class' => ['class-string|false', 'object='=>'object'],
+'get_class_methods' => ['array<int,string>|null', 'class'=>'mixed'],
 'get_class_vars' => ['array<string,mixed>', 'class_name'=>'string'],
 'get_current_user' => ['string'],
 'get_declared_classes' => ['array<int,class-string>'],
 'get_declared_interfaces' => ['array<int,class-string>'],
-'get_declared_traits' => ['array<int,class-string>'],
+'get_declared_traits' => ['array<int,class-string>|null'],
 'get_defined_constants' => ['array<string,int|string|float|bool|null|array|resource>', 'categorize='=>'bool'],
 'get_defined_functions' => ['array<string,array<string,callable-string>>', 'exclude_disabled='=>'bool'],
 'get_defined_vars' => ['array'],
-'get_extension_funcs' => ['array<int,callable-string>', 'extension_name'=>'string'],
+'get_extension_funcs' => ['array<int,callable-string>|false', 'extension_name'=>'string'],
 'get_headers' => ['array|false', 'url'=>'string', 'format='=>'int', 'context='=>'resource'],
 'get_html_translation_table' => ['array', 'table='=>'int', 'flags='=>'int', 'encoding='=>'string'],
 'get_include_path' => ['string'],
 'get_included_files' => ['array<int,string>'],
 'get_loaded_extensions' => ['array<int,string>', 'zend_extensions='=>'bool'],
-'get_magic_quotes_gpc' => ['false'],
-'get_magic_quotes_runtime' => ['false'],
+'get_magic_quotes_gpc' => ['int|false'],
```

Comment:
https://www.php.net/manual/en/function.get-magic-quotes-gpc.php#refsect1-function.get-magic-quotes-gpc-changelog

This always returns false as of 5.4.0 - Phan only supports call maps for 5.6.0 at earliest, deliberately.